daemon/daemon.go

```diff
@@ -19,22 +24,45 @@ type Daemon struct {
 	UserServer    *grpc.Server
 	ControlServer *grpc.Server
 
-	version string
-	runtime *runtime.Runtime
-	pool    map[string]*DriverPool
-	mutex   sync.Mutex
+	version   string
+	runtime   *runtime.Runtime
+	driverEnv []string
+	pool      map[string]*DriverPool
+	mutex     sync.Mutex
```

Comment:
Conventionally, a mutex goes before the fields it protects. If that's not the convention we're using here, please include a comment to say which fields it protects.

Separately: I recommend calling this more simply `mu` (which is a fairly common convention for a mutex field).

Reply:
Oh, sure, I haven't yet refactored it. Both comments make perfect sense.